Shared/Samples/Identify layer features/IdentifyLayerFeaturesView.swift

```diff
@@ -0,0 +1,182 @@
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct IdentifyLayerFeaturesView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The tapped screen point.
+    @State var tapScreenPoint: CGPoint!
+    
+    var body: some View {
+        ZStack {
+            MapViewReader { proxy in
+                MapView(map: model.map)
+                    .onSingleTapGesture { screenPoint, _ in
+                        tapScreenPoint = screenPoint
+                    }
+                    .task {
+                        do {
+                            // Add map image layer.
+                            let mapImageLayer = ArcGISMapImageLayer(url: .worldCities)
+                            try await mapImageLayer.load()
+                            
+                            // Hide continent and world layers.
+                            mapImageLayer.subLayerContents[1].isVisible = false
+                            mapImageLayer.subLayerContents[2].isVisible = false
+                            model.map.addOperationalLayer(mapImageLayer)
+                            
+                            // Add feature layer.
+                            let featureTable = ServiceFeatureTable(url: .damageAssessment)
+                            try await featureTable.load()
+                            let featureLayer = FeatureLayer(featureTable: featureTable)
+                            model.map.addOperationalLayer(featureLayer)
+                        } catch {
+                            model.error = error
+                        }
+                    }
+                    .task(id: tapScreenPoint) {
+                        // Identify layers using the screen point.
+                        if let screenPoint = tapScreenPoint,
+                           let results = try? await proxy.identifyLayers(
+                            screenPoint: screenPoint,
+                            tolerance: 12,
+                            returnPopupsOnly: false,
+                            maximumResultsPerLayer: 10) {
+                            model.handleIdentifyResults(results)
+                            model.isShowingLayerAlert = true
+                        }
+                    }
+                    .alert(isPresented: $model.isShowingLayerAlert) {
+                        Alert(
+                            title: Text(model.layerAlertTitle),
+                            message: Text(model.layerAlertMessage)
+                        )
+                    }
+                    .alert(isPresented: $model.isShowingErrorAlert, presentingError: model.error)
+            }
+        }
+    }
+}
+
+private extension IdentifyLayerFeaturesView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A map with a topographic basemap centered on the United States..
+        let map: Map = {
```

Comment:
Given that the model is only holding 1 ArcGIS object, I'd like to get rid of the model object and move everything into the view. A bit of refactoring of this sample would be great. (methods can be put in an extension to avoid clutter)